src/java/org/apache/cassandra/concurrent/ExecutionFailure.java

```diff
@@ -49,6 +51,9 @@ public static void handle(Throwable t)
     {
         try
         {
+            if (t instanceof RequestTimeoutException || t instanceof CancellationException)
```

Comment:
i believe we talked about this before, this is to avoid logging `CancellationException` right?  is `RequestTimeoutException` getting logged or something?

Reply:
Yes, reported twice I think, once by the `Future` we use to report the error, and once by the `AccordAgent`.